### ISSUE TITLE
pip install gelfclient failed

### ISSUE THREAD

**Issue by renardguill** [orionvm/python-gelfclient#2]
Hi,
on pip install, I have :

```
        from client import UdpClient
    ImportError: No module named 'client'
```

But when I change `from client import UdpClient` by `from .client import UdpClient` (add dot before client) in `__init__.py` it's working.

**Comment by Teumer** [comment]
Thanks, this resolved my issue as well.

**Comment by shizacat** [comment]
+, and me helped

**Comment by mcclymont** [comment]
Thanks, should be fixed and python3 compatible with version 0.0.5
